Challenge-4/Bot_2/src/helper.spec.ts

```diff
@@ -0,0 +1,133 @@
+import { MockEthersProvider } from "forta-agent-tools/lib/test";
+import { ethers } from "forta-agent";
+import Helper from "./helper";
+import { createAddress } from "forta-agent-tools";
+
+const mockAbi = [
+  "function getConfiguration(address cometProxy) view returns (tuple(address governor, address pauseGuardian, address baseToken, address baseTokenPriceFeed, address extensionDelegate, uint64 supplyKink, uint64 supplyPerYearInterestRateSlopeLow, uint64 supplyPerYearInterestRateSlopeHigh, uint64 supplyPerYearInterestRateBase, uint64 borrowKink, uint64 borrowPerYearInterestRateSlopeLow, uint64 borrowPerYearInterestRateSlopeHigh, uint64 borrowPerYearInterestRateBase, uint64 storeFrontPriceFactor, uint64 trackingIndexScale, uint64 baseTrackingSupplySpeed, uint64 baseTrackingBorrowSpeed, uint104 baseMinForRewards, uint104 baseBorrowMin, uint104 targetReserves, tuple(address asset, uint8 decimals, uint256 conversionFactor)[] assetConfigs) configuration)",
+  "function getUtilization() public view returns (uint)",
+  "function getSupplyRate(uint utilization) public view returns (uint64)",
+  "function getBorrowRate(uint utilization) public view returns (uint64)",
+];
+
+describe("Helper class test suite", () => {
+  let mockProvider: MockEthersProvider;
+  let Iface: ethers.utils.Interface = new ethers.utils.Interface(mockAbi);
+  let helper: Helper;
+  const usdcTokenAddress = "0xc3d688B66703497DAA19211EEdff47f25384cdc3";
+  let mockConfigProxy = "0x316f9708bb98af7da9c68c1c3b5e79039cd336e3";
```

Comment:
Could create more simple mock addresses for these as well. Using `createAddress()` like you used elsewhere.